client/src/pages/Notifications/useStyles.ts

```diff
@@ -0,0 +1,56 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  typography: {
+    padding: theme.spacing(2),
+    fontSize: 15,
+    fontWeight: 'bold',
+    paddingBottom: '0',
+  },
+  poptitle: {
+    marginTop: 20,
```

Comment:
You can remove this? As you are setting the margin to 20 in the next line

Reply:
removed in the recent commit